src/ContainerFactory.ts

```diff
@@ -101,6 +102,8 @@ export function createContainer(): Container {
             sessionConfig,
             companyAuthFeatureEnabled));
 
+    container.bind(FeatureToggleMiddleware).toConstantValue(FeatureToggleMiddleware);
```

Comment:
That does not make much sense and I believe it is not needed cause you are not injecting instances of `FeatureToggleMiddleware` due to required argument.

By the way I'd love to have separate PR that refactors feature toggle after one for file upload was removed. That would keep PR small and focus to me. I am not sure through how much work it would be to split it now, if too much then not a point.